### PR TITLE
Add address, postcode, town state and country to locations

### PR DESCRIPTION
**Pull request by trasher**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes

### CODE REVIEW

File: install/update_92_93.php [hunk 1]
```diff
@@ -621,6 +621,53 @@ function update92to93() {
 
    /** /Datacenters */
 
+   /** Add address to locations */
+   if (!$DB->fieldExists('glpi_locations', 'address')) {
+      $migration->addField(
+         'glpi_locations',
+         'address',
+         'text COLLATE utf8_unicode_ci',
```
Comment (wawax) [inline install/update_92_93.php]: you can just use 'text', add migration::fieldFormat will take care of collection and default value

File: install/update_92_93.php [hunk 2]
```diff
@@ -621,6 +621,53 @@ function update92to93() {
...
+         ['after' => 'sons_cache']
+      );
+   }
+
+   if (!$DB->fieldExists('glpi_locations', 'postcode')) {
+      $migration->addField(
+         'glpi_locations',
+         'postcode',
+         'varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL',
```
Comment (wawax) [inline install/update_92_93.php]: you can replace the line by 'string'

**Comment by trasher** [comment]
Changes done, thank you

**Comment by tsmr** [comment]
Add these new fields into Tooltip too ? it's, i think, a good idea no ?

**Comment by trasher** [comment]
@tsmr yes :) @wawax already ask me to add that; I'm working on it ;)

BTW, I was going to add only address, town and country in the tooltip. I'm pretty sure the postal code is not relevant; but I'm not sure for the state.

**Comment by trasher** [comment]
capture d ecran de 2018-01-04 15-08-46

**Pull request merged**